WPF/Practise_02.01/2020.02.01/MainWindow.xaml

```diff
@@ -0,0 +1,95 @@
+﻿<Window x:Class="_2020._02._01.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:_2020._02._01"
+        xmlns:model="clr-namespace:_2020._02._01.Model"
+        xmlns:viewModel="clr-namespace:_2020._02._01.ViewModel"
+        mc:Ignorable="d"
+        Title="MainWindow" Height="450" Width="800">
+    <Window.Resources>
+        <DataTemplate x:Key="CarTemplate" DataType="{x:Type model:Car}">
+            <StackPanel Orientation="Horizontal">
+                <Image Source="{Binding ImagePath}"
+                       Width="70"
+                       Height="50"></Image>
+                <Label Content="{Binding Brand}"></Label>
+                <Label Content="{Binding Model}"></Label>
+            </StackPanel>
+        </DataTemplate>
```

Comment:
It looks better when we use simple tag where possible:
<Label Content="{Binding Brand}" /> instead of
<Label Content="{Binding Brand}"></Label>